.github/workflows/pylint.yaml

```diff
@@ -0,0 +1,28 @@
+name: Lint python code
+
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches: '**'
+
+jobs:
+  pylint:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Checkout code including full history and submodules
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+
+    - name: Install dependencies
+      run: |
+          python -m pip install --upgrade pip
+          pip install pylint -r tests/integration/requirements.txt
+
+    - name: Execute pylint
+      run: |
+          python -c "import sys; print(sys.version)"
+          pylint tests
```

Comment:
All of this file should be part of `compliance.yaml`.

Reply:
ok, I'll move the python linting into this file